apps/fees-pay/ccpay-service-request-cpo-update-service/demo-image-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: ccpay-service-request-cpo-update-service
```

Comment:
Please update name to correct value `demo-service-request-cpo-update-service`

Reply:
This has now been updated as per the comment.